model-server/src/test/kotlin/org/modelix/model/server/handlers/ModelReplicationServerTest.kt

```diff
@@ -182,13 +184,21 @@ class ModelReplicationServerTest {
     @Test
     fun `responds with 400 when deleting from an invalid repository ID`() {
         runWithTestModelServer { _, _ ->
+            val client = createClient { install(ContentNegotiation) { json() } }
+
             val response = client.delete {
                 url {
                     appendPathSegments("v2", "repositories", "invalid with spaces", "branches", "master")
                 }
             }
-            assertEquals(HttpStatusCode.BadRequest, response.status)
-            assertContains(response.bodyAsText(), "Invalid repository name")
+
+            val expectedProblem = Problem(
+                type = "/problems/invalid-repository-id",
+                title = "Invalid repository ID.",
+                status = 400,
```

Comment:
Not sure if this is better?

```suggestion
                status = HttpStatusCode.BadRequest.value,
```